src/components/ContentBook.js

```diff
@@ -1,16 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBookAction } from './redux/books/books';
 
-const ContentBook = ({ title, author }) => (
-  <li>
-    <h3>{title}</h3>
-    <p>{author}</p>
-    <button type="button">Delete</button>
-  </li>
-);
+const ContentBook = ({ book }) => {
+  const dispatch = useDispatch();
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    dispatch(removeBookAction(book.id));
+  };
+
+  return (
+    <li>
+      <h3>{book.title}</h3>
+      <p>{book.author}</p>
+      <button type="submit" onClick={handleSubmit}>Remove</button>
+    </li>
+  );
+};
 ContentBook.propTypes = {
-  title: PropTypes.string.isRequired,
-  author: PropTypes.string.isRequired,
+  /*eslint-disable */ 
```

Comment:
Good job adding redux :clap: , However, we should always try not to disable any linters as, a linter is a tool that analyzes our source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. So I suggest removing this line and implement it like below :point_down: 

```
ContentBook.propTypes = {
  book: PropTypes.shape({
    title: PropTypes.string,
    author: PropTypes.string,
    id: PropTypes.string,
  }).isRequired,
};

```